quarto/_extensions/countdown/assets/countdown.css

```diff
@@ -0,0 +1,169 @@
+.countdown {
+  --_running-color: var(--countdown-color-running-text, rgba(0, 0, 0, 0.8));
+  --_running-background: var(--countdown-color-running-background, #43AC6A);
+  --_running-border-color: var(--countdown-color-running-border, rgba(0, 0, 0, 0.1));
+  --_finished-color: var(--countdown-color-finished-text, rgba(0, 0, 0, 0.7));
```

Comment:
I started thinking about how we'd replace prismatic and these lines kind of hint at where my head is at.

First, we definitely do not need a prismatic replacement in the lua filter for this PR; we can certainly iterate after merging this one.

The set up with these lines is that we could tint or shade the countdown border and text color based on the background color. I went with shading (darkening) in the CSS, but we could have tinted (lightend) as well with `rgba(255, 255, 255, 0.1)` or similar.

The question is how we'd know whether we should lighten or darken. But I'm okay waiting on that as well and working it out in follow up PRs.

Reply:
Thinking out loud, there would be a Python port coming next. So, would either of these options be acceptable to avoid moving it into the Lua filter?

1. Write the lighting and darkening logic in JavaScript (preferred) 
2. Custom CSS properties using `calc` alongside a flag on the inline element? `light-color`/`dark-color`? 

```
:root {
      --main-background-color: #3498db;
      --text-color: calc(1 - var(--main-background-color) * 0.7); /* Default text color */
    }
```

Reply:
Yeah, I'd really like to handle it in CSS, because that will be the easiest in the long term. Hopefully sometime soon we'll get [color-contrast()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast).

For now, I'm okay with the current approach and we can work it out in future iterations.